python/lsst/analysis/tools/actions/plot/gridPlot.py

```diff
@@ -90,16 +103,49 @@ class GridPlot(PlotAction):
     def __call__(self, data: KeyedData, **kwargs) -> PlotResultType:
         """Plot data."""
         fig = plt.figure(figsize=self.figsize, dpi=self.dpi)
+        figureInfo = {"figsize": fig.get_size_inches(), "dpi": fig.get_dpi()}
+
+        if self.height_ratios is None:
+            height_ratios = np.ones(self.numRows) / self.numRows
+        else:
+            height_ratios = self.height_ratios / np.sum(self.height_ratios)
+
+        if self.width_ratios is None:
+            width_ratios = np.ones(self.numCols) / self.numCols
+        else:
+            width_ratios = self.width_ratios / np.sum(self.width_ratios)
+
+        includesPlotInfoPanel = False
+        addPlotInfoRow = 1
+        for panel in self.panels.values():
+            if isinstance(panel.plotElement, PlotInfoElement):
+                includesPlotInfoPanel = True
+                addPlotInfoRow = 0
+                break
+
+        if not includesPlotInfoPanel:
+            height_ratios = np.append([1.0], 20.0 * height_ratios)
```

Comment:
What's the purpose of this? Is it to reserve some small fraction of space at the top of a plot if a plot info is _not_ given?

Reply:
If so, please add a brief in-line comment above with your rationale. However, pending a discussion below of a potential re-write of how we handle plot info elements, this may end up disappearing in the rewrite.

Reply:
For now, this has been removed in lieue of developing a non-carve-out method of adding plotInfo if/when not explicitly specified.